src/processing/script/index.js

```diff
@@ -49,9 +49,12 @@ function preprocess (code) {
     return { bom, preprocessed };
 }
 
-function postprocess (processed, withHeader, bom) {
+function postprocess (processed, withHeader, bom, strictMode) {
+    // NOTE: If the 'use strict'; directive is not in the beginning of the file, it is ignored.
+    // As we insert our header in the beginning of the script, so we must to put a new 'use strict';
+    // before the header, otherwise it will be ignored.
```

Comment:
```
    // NOTE: If the 'use strict' directive is not in the beginning of the file, it is ignored.
    // As we insert our header in the beginning of the script, we must put a new 'use strict'
    // before the header, otherwise it will be ignored.
```